### PR TITLE
Simple solution for #365

### PR DESCRIPTION
**Pull request by janssenda-adobe**
Ignore the users option if set to default

### CODE REVIEW

**Comment by adorton-adobe** [comment]
I still get an error:

```
λ  py .\dist\user-sync.pex -c .\config\user-sync-config.yml --process-groups --users mapped -t --adobe-only-user-action remove --adobe-only-user-list .\adobe_only.csv
2018-07-27 18:48:52 15500 INFO config - Using main config file: .\config\user-sync-config.yml (encoding utf8)
2018-07-27 18:48:52 15500 CRITICAL main - You cannot specify both a --users arg and an --adobe-only-user-list arg
```

**Comment by janssenda-adobe** [comment]
I believe that is as intended, since --users was specified on the command line.  We could bypass any command line arg instead if you would prefer

File: user_sync/config.py [hunk 1]
```diff
@@ -186,10 +188,14 @@ def load_invocation_options(self):
...
-        elif options['users'] and options['adobe_only_user_list']:
-            raise AssertionException('You cannot configure both a default "users" option (%s) '
-                                     'and a default "adobe-only-user-list" option (%s)' %
-                                     (' '.join(options['users']), options['adobe_only_user_list']))
+
+        # Check both that options['users'] exists, AND that it is not the default value before checking against
+        # the adobe_only_user_list parameter -  otherwise, defaults cause exception to be thrown unconditionally when
+        # the user list option is specified
+        elif (options['users'] and options['users'] != self.invocation_defaults['users']) and options['adobe_only_user_list']:
+                raise AssertionException('You cannot configure both a default "users" option (%s) '
```
Comment (adorton-adobe) [inline user_sync/config.py]: @janssenda-adobe Will you please fix the indentation on this raise statement so it's just one level indented from the elif block?  It doesn't seem to be causing any issues, but a linter would probably complain about it, and we want the diff to render accurately.

Comment (janssenda-adobe) [reply user_sync/config.py]: @adorton-adobe Thanks! Fixed that

**Pull request merged**